File: jApp/src/main/java/j/s/yarlykov/ui/fragmentbased/DevInfoFragment.java
package j.s.yarlykov.ui.fragmentbased;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import j.s.yarlykov.R;

public class DevInfoFragment extends Fragment {
    public static DevInfoFragment create() {
        return new DevInfoFragment();
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.developer_fr, container, false);
    }
}

File: kApp/src/main/java/k/s/yarlykov/ui/fragmentbased/DevInfoFragment.kt
package k.s.yarlykov.ui.fragmentbased

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import k.s.yarlykov.R

class DevInfoFragment : Fragment() {

    companion object {
        fun create(): DevInfoFragment = DevInfoFragment()
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.developer_fr, container, false)
    }

}
